tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml

```diff
@@ -112,6 +112,7 @@ stages:
         set -e -x
         cp "$(Pipeline.Workspace)/ios_packaging_artifacts_full/pod-archive-onnxruntime-c-$(ortPodVersion).zip" swift/
         export ORT_IOS_POD_LOCAL_PATH="swift/pod-archive-onnxruntime-c-$(ortPodVersion).zip"
-        xcodebuild test -scheme onnxruntime -destination 'platform=iOS Simulator,name=iPhone 14'
+        # hardcode iOS 16.4 for now. latest macOS-13 image defaults to iOS 17 (beta).
+        xcodebuild test -scheme onnxruntime -destination 'platform=iOS Simulator,OS=16.4,name=iPhone 14'
```

Comment:
SPM repo is using macos-13 image agent too. assume I'll have to hardcode that to 16.4 as well?

Reply:
perhaps - I think this test didn't actually fail, but we don't want to be using a beta iOS version. that said, I'm working on a script to query the simulator info from xcode, so hopefully the hardcoded values can be replaced soon.

Reply:
Got it. thanks!